tests/test_routes.py

```diff
@@ -240,3 +240,40 @@ def test_create_order_bad_status(self):
         test_order["status"] = "created"  # wrong value
         response = self.app.post(BASE_URL, json=test_order)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    ######################################################################
+    #  ITEM - P L A C E   T E S T   C A S E S   H E R E
+    ######################################################################
+
+    def test_add_item(self):
+        """It should Add an item to an order"""
+        order = self._create_orders(1)[0]
+        item = OrderItemFactory()
+        resp = self.app.post(
+            f"{BASE_URL}/{order.id}/items",
+            json=item.serialize(),
+            content_type="application/json",
+        )
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+        data = resp.get_json()
+        logging.debug(data)
```

Comment:
debug statements are used for local testing only, so you can remove this line.

Reply:
done